ql/src/java/org/apache/hadoop/hive/ql/parse/repl/load/message/AlterDatabaseHandler.java

```diff
@@ -86,11 +97,45 @@
       // Only database object is updated
       updatedMetadata.set(context.dmd.getEventTo().toString(), actualDbName,
               null, null);
+      processSnapshots(context.hiveConf, oldDb, newDb);
       return Collections.singletonList(alterDbTask);
     } catch (Exception e) {
       throw (e instanceof SemanticException)
           ? (SemanticException) e
           : new SemanticException("Error reading message members", e);
     }
   }
+
+  private void processSnapshots(HiveConf conf, Database before,
```

Comment:
can use a common method. done at 2 places.

Reply:
Refactored to `SnapshotUtils`

Reply:
Moved to Snapshot Utils